Cargo.toml

```diff
@@ -14,7 +14,8 @@ authors = ["nabijaczleweli <nabijaczleweli@gmail.com>",
            "Cat Plus Plus <piotrlegnica@piotrl.pl>",
            "Liigo <liigo@qq.com>",
            "azyobuzin <azyobuzin@users.sourceforge.jp>",
-           "Peter Atashian <retep998@gmail.com>"]
+           "Peter Atashian <retep998@gmail.com>",
+           "Daniel Abramov <dabramov@snapview.de>",]
```

Comment:
What do you mean "reiterate"? It has never been mentioned in README that you don't want contributors to add changes there. I just derived the style based on the previous changes and used them to follow the common style of repository.